grid.py

```diff
@@ -238,8 +238,9 @@ def xy_to_grid(self, pos):
 
     def grid_to_xy(self, i):
         ''' Convert from grid index to sprite x,y. '''
-        return ((self.left + i % COL * self.xinc),
-                (self.top + (i / COL) * self.yinc))
+        x = (self.left + i % COL * self.xinc)
+        y = (self.top + (i // COL) * self.yinc)
```

Comment:
optionally, we could remove the redundant parenthesis (C0325).

Reply:
Thanks, fixed in d2a1bf1.